### ISSUE TITLE
Manque d'une erreur dans la modification de quizz (PUT)

### ISSUE THREAD

**Issue by LucieFournier** [romulusFR/lifap5-backend-2019-2020#14]
Le serveur peut renvoyer une erreur 500 quand on change par exemple le nom du quizz en un autre nom qui existait déjà (duplicate key value violates unique constraint \"quiz_title_key\").
![Erreur500PutQuizz](https://user-images.githubusercontent.com/60646360/79746523-696ec480-830a-11ea-8706-00ed2d4837b2.JPG)

**Comment by romulusFR** [comment]
Effectivement, ca devrait plutôt être <https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400> ou il y a autre chose ? @ecoquery m'en a parlé

**Comment by Webcretaire** [comment]
C'est pas 409 actuellement quand on a cette erreur dans un post et pas un put ? (il me semble de mémoire)

**Comment by LucieFournier** [comment]
Si effectivement c'est une 409 dans un POST mais ça me renvoie une 500 dans un PUT
